src/main/java/com/openshift/internal/restclient/capability/resources/PodExec.java

```diff
@@ -0,0 +1,156 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.openshift.internal.restclient.capability.resources;
```

Comment:
Please repackage here: com.openshift.internal.restclient.api.capabilities  as I'm trying to align more with how resources are discovered from the server.